templates/admin/usage_dashboard.html

```diff
@@ -0,0 +1,42 @@
+{% extends "/base.html" %}
+{% set active_page = "profile" %}
+{% set active_link = "admin" %}
+
+{% block content %}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css">
+<script src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script>
+<style>
+    .ct-bar {
+        stroke: #aae266 !important;
+    }
+    .ct-line, .ct-point {
+        stroke: #3AB0D5 !important;
+    }
+</style>
+<div class="container">
+    <div class="col-md-12" style="margin-top:30px;">
+        <h1><strong>{{ _('Usage Dashboard') }}</strong></h1>
+    </div>
+</div>
+<div class="container">
+    <div class="col-md-12">
+            <h2>{{ _('Platform Totals (Last ' + months + ' Months)' )}}</h2>
+            <table class="table table-striped">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th><p><strong>Last 6 Months</strong></p></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for stat, value in stats.items() %}
+                    <tr>
+                        <th><p><strong>{{ stat | replace('_', ' ') }}</strong></p></th>
```

Comment:
If `stat` returns a key value pair, per comment [here](https://github.com/bloomberg/pybossa/pull/824#discussion_r1131570370), this could change to be:

```html
<th><p><strong>Projects using {{ stat }}</strong></p></th>
```

We may also want to consider removing "Projects using" altogether, as it may be redundant. However, could we show the value in a mouseover tooltip or on an info icon?

```html
<th>
  <p><strong>{{ stat }}</strong></p>
  <div>
     <i class="fa-regular fa-circle-info" title={{ stats[stat] }}></i>
  </div>
</th>
```

Reply:
Since not all table rows start with "Projects using", this could add unnecessary complexity. I am on board with removing "Projects using" altogether for simplicity.